pkg/api/v1alpha1/condition_consts.go

```diff
@@ -0,0 +1,49 @@
+package v1alpha1
+
+// Conditions and condition Reasons for the Cluster object.
+const (
+	// ReadyCondition reports a summary of other conditions, indicating an overall operational
+	// state of the cluster.
+	ReadyCondition ConditionType = "Ready"
+
+	// PendingUpdateReason reports when an update has been scheduled but has yet to happen.
+	PendingUpdateReason = "PendingUpdate"
+
+	// ControlPlaneInitializedCondition reports that all the control plane nodes are
+	// in a fully ready and running state.
+	ControlPlaneReadyCondition ConditionType = "ControlPlaneReady"
+
+	// ControlPlaneInitializedCondition reports that the first control plane instance has been initialized
+	// and so the control plane is available and an API server instance is ready for processing requests.
+	ControlPlaneInitializedCondition ConditionType = "ControlPlaneInitialized"
+
+	// WaitingForControlPlaneInitializedReason used when cluster is waiting for control plane to be initialized.
+	WaitingForControlPlaneInitializedReason = "WaitingForControlPlaneInitialized"
```

Comment:
what is this reason telling us? really nothing right? what about `ControlPlaneInitializationInProgressReason`